rest/blip_sync.go

```diff
@@ -297,7 +297,7 @@ func (bh *blipHandler) handleSubChanges(rq *blip.Message) error {
 func (bh *blipHandler) sendChanges(sender *blip.Sender, params *subChangesParams) {
 	defer func() {
 		if panicked := recover(); panicked != nil {
-			base.Warn("[%s] PANIC sending changes: %v\n%s", bh.blipContext.ID, panicked, debug.Stack())
+			base.WarnR("[%s] PANIC sending changes: %v\n%s", base.MD(bh.blipContext.ID), base.MD(panicked), debug.Stack())
```

Comment:
Likewise, ideally we wouldn't redact the return value of panicked as it would only be a type of error, rather than contain any data.

Reply:
Fixed